samples/widgets/sales-dashboard-widgets/RevenueByProductWithUserPreferences/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Year should be 2018

Reply:
fixed in df8f0f0400a8f4e346e204d693a3f7abded71985